rfcs/res/scaling_dependencies.dot

```diff
@@ -0,0 +1,53 @@
+digraph G {
+  rankdir=LR;
+
+  // Protocol Aspects
+  {
+    node [color=blue, shape=square, width=2.2, fixedsize=true];
+    ScanSize [label="Scan State Size (+)"];
+    SlotTime [label="Slot Time (-)"];
+  }
+
+  // Actionable Dependencies
+  {
+    node [shape=rectangle];
+    ELSync [label="Epoch Ledger Sync Time (-)"];
+    ELMem [label="Epoch Ledger Memory (-)"];
+    StLDiffApp [label="Staged Ledger Diff Application Time (-)"];
+    SnLCommit [label="Snarked Ledger Commit Time (-)"];
+    ScanSyncTime [label="Scan State Sync Time (-)"];
+    ScanMem [label="Scan State Memory (-)"];
+    ScanSyncSlotDivorce [label="Scan State Sync Slot Divorce"];
+    HashTime [label="Poseidon Hash Time (-)"];
+    VRFTime [label="VRF Time (-)"];
+  }
+
+  // Goals
+  {
+    rank=same;
+    node [shape=oval];
+    NumAccs [label="# of Accounts"];
+    TPS [label="TPS"];
+    Latency [label="Block Latency"];
+    Delegation [label="# of Delegators on one Account"];
+  };
+
+  // Positive Arcs
+  {
+    edge [color=black];
+    {ScanSyncTime, ScanSyncSlotDivorce, HashTime} -> ScanSize
+    {StLDiffApp, ScanSyncSlotDivorce, SnLCommit, HashTime} -> SlotTime;
+    HashTime -> StLDiffApp;
+    {ELSync, ELMem, SnLCommit} -> NumAccs;
+    {ScanSize, ScanMem, SlotTime} -> TPS;
+    SlotTime -> Latency;
+    VRFTime -> Delegation;
+  }
+
+  // Negative Arcs
+  {
+    edge [color=red];
+    ScanSize -> {SnLCommit, StLDiffApp, Latency};
```

Comment:
Shouldn't these be black arcs?

Reply:
As the scan state size increases, it takes longer to commit the snarked ledger since the # of txns in a single proof emitted from the scan state is larger, it takes longer to apply a staged ledger diff since the # of txns that can be included in a single diff increases, and the # of blocks you have to wait until a proof is emitted also goes up (unless the total snark work goes up by the same amount as the size increase).

Reply:
The "desired properties" being compared had me confused.